### PR TITLE
GAS Job Restrictions

### PR DESCRIPTION
**Pull request by IvanIvanovIvanovsky**
Allows all GAS to be civilians now that they are citizens under the lore. Adds Bartender and Chef to grade A, as before it was only grade B. This is my first PR in over six months so I'm keeping it simple. Playtested, looks like it works fine to me. May create another more expansive PR for reworking the shitcode that is GAS grading. I'd like feedback on what jobs you think GAS should or should not be in so feel free to comment it here incase I do make that PR.

### CODE REVIEW

**Comment by Kaksisilma** [comment]
Idk about passenger GAS but eh. Additions to A are fine.

Also:
Capture

File: maps/torch/job/torch_jobs.dm [hunk 1]
```diff
@@ -57,6 +57,10 @@
 /decl/cultural_info/culture/nabber/New()
 	LAZYADD(valid_jobs, /datum/job/scientist_assistant)
 	..()
+	
+	/decl/cultural_info/culture/nabber/New()
```
Comment (Kaksisilma) [inline maps/torch/job/torch_jobs.dm]:
```suggestion
/decl/cultural_info/culture/nabber/New()
```

**Review by Kaksisilma — CHANGES_REQUESTED** [review]
This will probably fix it.

**Comment by IvanIvanovIvanovsky** [comment]
Once I downloaded a copy and it didn't work I edited it in DM until it did, to save time, and so I copied the changes by memory to the github version, seems like I made an error.

**Comment by Vizoplez** [comment]
Good for merge if @Kaksisilma agrees

**Comment by benj8560** [comment]
wanted to suggest giving them prospecting. Always kinda made sense they would suit that kinda work. They got RIGs now and their strength makes them well suited for any labor job. I believe prospectors already get EVA so them getting their suits is already taken care of.
also no doubt some funny memes will come of having a GAS in a small enclosed space on the Gaunt *chitter

**Comment by Kaksisilma** [comment]
> wanted to suggest giving them prospecting. Always kinda made sense they would suit that kinda work. They got RIGs now and their strength makes them well suited for any labor job. I believe prospectors already get EVA so them getting their suits is already taken care of. also no doubt some funny memes will come of having a GAS in a small enclosed space on the Gaunt *chitter

Sounds good on paper but on practice I think it'll be a nightmare.

Code-wise, this PR is OK so I'll go ahead and merge it. If push comes to shove we can remove passenger GAS later.

**Pull request merged**